$OpenBSD: patch-mk_bootstrap_mk,v 1.3 2004/05/20 00:08:22 dons Exp $
Make sure we know how to build things. I don't know why this isn't
needed on, e.g. linux

--- mk/bootstrap.mk.orig	Wed Feb  4 21:05:04 2004
+++ mk/bootstrap.mk	Thu May 20 09:49:39 2004
@@ -8,6 +8,8 @@
 # compile the .hc files, so we have to duplicate that functionality here.
 # The result is unfortunately ugly, but we don't have another choice.
 
+include $(TOP)/mk/suffix.mk
+
 TOP_SAVED := $(TOP)
 TOP:=$(TOP)/ghc
 
